src/main/java/com/guciowons/shoppingbasket/Basket/BasketController.java

```diff
@@ -19,7 +19,7 @@ public ResponseEntity<Basket> createBasket(){
     }
 
     @PutMapping(value = "/{basketId}/{productId}/{quantity}")
-    public ResponseEntity addProductToBasket(@PathVariable int basketId, @PathVariable int productId, @PathVariable int quantity){
+    public ResponseEntity<Object> addProductToBasket(@PathVariable int basketId, @PathVariable int productId, @PathVariable int quantity){
```

Comment:
Jak już chcesz podać typ to bądź precyzyjny, powinno być `ResponseEntity<Basket>`